doc/classes/Tween.xml

```diff
@@ -91,6 +91,7 @@
 		[url=https://raw.githubusercontent.com/godotengine/godot-docs/master/img/tween_cheatsheet.webp]Tween easing and transition types cheatsheet[/url]
 		[b]Note:[/b] Tweens are not designed to be re-used and trying to do so results in an undefined behavior. Create a new Tween for each animation and every time you replay an animation from start. Keep in mind that Tweens start immediately, so only create a Tween when you want to start animating.
 		[b]Note:[/b] The tween is processed after all of the nodes in the current frame, i.e. node's [method Node._process] method would be called before the tween (or [method Node._physics_process] depending on the value passed to [method set_process_mode]).
+		[b]Note:[/b] When creating a [Tween] by using [method SceneTree.create_tween] the [Tween] will be detached from the actual [Node] and it won't be killed when the [Node] is freed. Prefer using [method Node.create_tween] if you want the [Tween] to be automatically killed when the [Node] is freed.
```

Comment:
This is not entirely true. When a Node is freed, PropertyTweeners that target it will end automatically and Tween will kill itself if it has no Tweeners to animate. I think if Callable's `get_object()` return value is reliable, we could add a similar safeguard to MethodTweener and CallbackTweener to prevent errors.
By "not entirely true" I mean that Tween will continue to animate, but it will finish if there is nothing to animate (this is mostly relevant if you are also animating other objects).

Also shouldn't this note be in `SceneTree.create_tween()` description instead?

Reply:
@KoBeWi I've rewritten the note and moved it to `SceneTree` class reference file